public/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Freedomvote</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="navigation">
+      <nav class="navigation__nav">
+        <a href="#">
+          <img class="navigation__nav__logo" src="/img/logo.svg" />
```

Comment:
Dont nest to much. `.navigation__nav__logo` is okay but not necessary. I would prefer `.navigation__logo`